kibana/Dockerfile

```diff
@@ -1,19 +1,33 @@
-FROM openshift/base-centos7
+FROM openshift/base-centos7 
 
 MAINTAINER Eric Wolinetz <ewolinet@redhat.com>
 
-ENV KIBANA_VER 4.1.1
+EXPOSE 5601
```

Comment:
Do we want to expose port 5601 from the container?  Currently we let the template handle this and it maps it to port 80